Src/Base/AMReX_MultiFab.H

```diff
@@ -705,7 +705,25 @@ public:
     void AverageSync (const Periodicity& period = Periodicity::NonPeriodic());
     //! Sync up nodal data with weights
     void WeightedSync (const MultiFab& wgt, const Periodicity& period = Periodicity::NonPeriodic());
-    //! Sync up nodal data with owners overriding non-owners
+    /**
+     * \brief Synchronize nodal data.  The synchronization will override
+     * valid regions by the intersecting valid regions with a higher
+     * precedence.  The smaller the global box index is, the higher
+     * precedence the box has.  With periodic boundaries, for cells in the
+     * same box, those near the lower corner have higher precedence than
+     * those near the upper corner.
```

Comment:
@WeiqunZhang 

Does this suggestion look good to you? If not, could you edit it in my comment? I would like to have the green light from you before committing this change, so that I don't waste CI resources:


```suggestion
     * \brief Synchronize nodal data.  The synchronization will override
     * valid regions by the intersecting valid regions with a higher
     * precedence, according to the owner mask provided as input.
```